Server/src/main/java/org/xdi/oxauth/service/SessionIdService.java

```diff
@@ -143,9 +143,11 @@ public SessionId assertAuthenticatedSessionCorrespondsToNewRequest(SessionId ses
 	                log.info("Acr is changed. Session acr: " + sessionAcr + "(level: " + sessionAcrLevel + "), " +
 	                        "current acr: " + acrValue + "(level: " + currentAcrLevel + ")");
 	                
-	                // Requested acr method which not enabled
+	                // Requested acr method not enabled
 	                if (currentAcrLevel == null) {
-	                    throw new AcrChangedException();
+                        AcrChangedException ex = new AcrChangedException();
+                        ex.setMethodEnabled(false);
+                        throw ex;
```

Comment:
Does it make sense to add constructor that allows to specify flag=true/false ?

Then we will have:
```
throw new AcrChangedException(false);
```
instead of
```
AcrChangedException ex = new AcrChangedException();
ex.setMethodEnabled(false);
throw ex;
```